src/diagonal.works/b6/api/functions/functions.go

```diff
@@ -20,12 +20,13 @@ var functions = api.FunctionSymbols{
 	// collections
 	"collection":   collection,
 	"count-values": countValues,
+	"export":       export,
 	"filter":       filter,
 	"flatten":      flatten,
+	"histogram":    histogram,
 	"sum-by-key":   sumByKey,
 	"take":         take,
 	"top":          top,
-	"histogram":    histogram,
```

Comment:
SORRY I MISSED u were upholding order in here w last cl, we should lint no tho i think we d have to write our own :(